parser.go

```diff
@@ -24,7 +24,7 @@ type Index interface {
 	Indexes() dialect.Indexes
 }
 
-func (dm *DDLMaker) parse() error {
+func (dm *DDLMaker) parse() {
```

Comment:
There's no possibility that this `parse` method returns an error. So, the returned type `error` is removed.